demo/server/src/modules/passport-auth/index.ts

```diff
@@ -22,6 +22,9 @@ export function init(app: express.Express) {
   const userService: UserService = new SampleUserService();
   const authService: PassportAuth = new PassportAuth(userRepo, userService);
   authService.init(app, sessionOpts);
+  authService.accessDenied = function(req: express.Request, res: express.Response) {
```

Comment:
That's considered a hack in typescript (basically they way to do it in js)
Proper way to extend method is to create new class.